core/build.gradle.kts

```diff
@@ -68,7 +68,7 @@ kotlin {
         nodejs {
             testTask {
                 useMocha {
-                    timeout = "5s"
+                    timeout = "30s"
```

Comment:
The highest recorded test duration value was 8.6s, so probably 10-15 s should be fine

Reply:
Sure, I just don't see the point in this timeout at all. Don't understand what it brings to the table. So, I increased it to the point where it will most certainly not bother us. Do you have some ideas about why this limit could be useful? If so, maybe it makes sense to add it to the other targets as well?